src/views/2_SessionComponent.tsx

```diff
@@ -35,14 +35,23 @@ export class SessionComponent extends React.Component<Props, {}> {
                           hasLocusOfAttention={this.props.isActive && index === this.props.session.jobs.length - 1}/>
         );
 
+        let shutter: any;
+        if (!this.props.isActive) {
+          shutter = <div className="shutter" style={css.sessionShutter}></div>;
+        }
```

Comment:
Let's simplify this flow and always render the shutter, but change it's opacity based on `isActive`.